src/pages/MonthlyGood/MonthlyGoodLandingPage.vue

```diff
@@ -71,23 +71,27 @@
 			</template>
 		</kiva-as-expert>
 		<more-about-kiva />
-		<frequently-asked-questions />
+		<div class="kv-tailwinds">
```

Comment:
Refactoring this whole page for tailwinds was out of scope, so this just makes sure the faq component can use the tailwinds classes

Reply:
Should be `kv-tailwind`

Reply:
👍